web-ui/src/app/components/block-details/block-details.component.ts

```diff
@@ -26,7 +36,9 @@ export class BlockDetailsComponent implements OnInit {
     private errorService: ErrorService) { }
 
   ngOnInit() {
-    this.route.params.forEach(params => this.onBlockhash(params['query']));
+    this.blockhash = this.route.snapshot.paramMap.get('query');
```

Comment:
Why are you changing this?

Load a block on the UI and click the next or previous block links to see why the previous line is required.